tendermint/src/lite/verifier.rs

```diff
@@ -726,21 +726,21 @@ mod tests {
         let ts = &init_trusted_state(vac.clone(), vec![0, 1, 2, 3], 1);
 
         // 100% overlap (all signers present)
-        assert_single_ok(ts, vac.clone());
+        assert_single_ok(ts, vac);
```

Comment:
Nice, as a side-effect we could get rid of these clones 👍

Reply:
yes, makes it cleaner!